File: datasets/php-keep-code_davinci_001_temp_0.2-0/HumanEval_51_remove_vowels.php
<?php
// remove_vowels is a function that takes string and returns string without vowels.
// >>> remove_vowels('')
// ''
// >>> remove_vowels('abcdef')
// 'bcdf'
// >>> remove_vowels('aaaaa')
// ''
// >>> remove_vowels('aaBAA')
// 'B'
// >>> remove_vowels('zbcd')
// 'zbcd'
function remove_vowels($text) {
    $vowels = array('a', 'e', 'i', 'o', 'u');
    $text = str_replace($vowels, '', $text);
    return $text;
}

function candidate(...$args) {
    return remove_vowels(...$args);
}

function test(): void {
    if (candidate('') !== '') { throw new Exception("Test failed!"); }
    if (candidate("abcdef
ghijklm") !== "bcdf
ghjklm") { throw new Exception("Test failed!"); }
    if (candidate('fedcba') !== 'fdcb') { throw new Exception("Test failed!"); }
    if (candidate('eeeee') !== '') { throw new Exception("Test failed!"); }
    if (candidate('acBAA') !== 'cB') { throw new Exception("Test failed!"); }
    if (candidate('EcBOO') !== 'cB') { throw new Exception("Test failed!"); }
    if (candidate('ybcd') !== 'ybcd') { throw new Exception("Test failed!"); }
}

test();

File: datasets/scala-keep-code_davinci_001_temp_0.2/HumanEval_51_remove_vowels.scala
import scala.math._
import scala.collection.mutable._
object Problem {
    // remove_vowels is a function that takes string and returns string without vowels.
    // >>> remove_vowels('')
    // ''
    // >>> remove_vowels('abcdef')
    // 'bcdf'
    // >>> remove_vowels('aaaaa')
    // ''
    // >>> remove_vowels('aaBAA')
    // 'B'
    // >>> remove_vowels('zbcd')
    // 'zbcd'
    def removeVowels(text : String) : String = {
        var result = ""
        for (i <- 0 until text.length) {
            if (text(i) != 'a' && text(i) != 'e' && text(i) != 'i' && text(i) != 'o' && text(i) != 'u') {
                result += text(i)
            }
        }
        return result

    }
    def main(args: Array[String]) = {
    assert(removeVowels(("")).equals(("")));
    assert(removeVowels(("abcdef\nghijklm")).equals(("bcdf\nghjklm")));
    assert(removeVowels(("fedcba")).equals(("fdcb")));
    assert(removeVowels(("eeeee")).equals(("")));
    assert(removeVowels(("acBAA")).equals(("cB")));
    assert(removeVowels(("EcBOO")).equals(("cB")));
    assert(removeVowels(("ybcd")).equals(("ybcd")));
    }

}
